### ISSUE TITLE
CI failures in old code

### ISSUE THREAD

**Issue by jajhall** [ERGO-Code/HiGHS#2002]
home/runner/work/HiGHS/HiGHS/src/lp_data/HighsIis.cpp:43:70: warning: format specifies type 'int' but the argument has type 'HighsInt' (aka 'long') [-Wformat]
  for (HighsInt iCol = 0; iCol < num_iis_col; iCol++) printf("%9d ", iCol);
                                                              ~~~    ^~~~
                                                              %9ld
/home/runner/work/HiGHS/HiGHS/src/lp_data/HighsInterface.cpp:45:32: error: no matching function for call to 'max'
  HighsInt local_row_min_nnz = std::max(lp.num_col_, 2);
                               ^~~~~~~~
/usr/bin/../lib/gcc/x86_64-linux-gnu/12/../../../../include/c++/12/bits/algorithmfwd.h:407:5: note: candidate template ignored: deduced conflicting types for parameter '_Tp' ('HighsInt' (aka 'long') vs. 'int')
    max(const _Tp&, const _Tp&);
    ^
/usr/bin/../lib/gcc/x86_64-linux-gnu/12/../../../../include/c++/12/bits/stl_algo.h:5756:5: note: candidate template ignored: could not match 'initializer_list<_Tp>' against 'HighsInt' (aka 'long')
    max(initializer_list<_Tp> __l, _Compare __comp)
    ^
/usr/bin/../lib/gcc/x86_64-linux-gnu/12/../../../../include/c++/12/bits/stl_algo.h:5746:5: note: candidate function template not viable: requires single argument '__l', but 2 arguments were provided
    max(initializer_list<_Tp> __l)
    ^
/usr/bin/../lib/gcc/x86_64-linux-gnu/12/../../../../include/c++/12/bits/algorithmfwd.h:412:5: note: candidate function template not viable: requires 3 arguments, but 2 were provided
    max(const _Tp&, const _Tp&, _Compare);
    ^
1 error generated.

**Comment by jajhall** [comment]
Only the first was in latest. Other is in [fix-1999](https://github.com/ERGO-Code/HiGHS/tree/fix-1999)

**Comment by jajhall** [comment]
Closed by #2003
